.ci/run-regression.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+iterations="$1"
+sleep_time="$2"
+
+PWD="$(pwd)"
+EXEC="$PWD/target/release/rosenpass"
+LOGS="$PWD/output/logs"
```

Comment:
Maybe `mktemp(1)` is better than an `output/` directory? Just a suggestion, I do not have a strong opinion about either approach.

Reply:
Since it's running in the CI, it's a fresh folder anyway.